### PR TITLE
fix: Guard before accessing VariableDeclarator init (#4) :bug:

### PR DESCRIPTION
**Pull request by mseeley**
The PR includes a guard to address #4.

### CODE REVIEW

File: test/fixtures/for-in-declaration-in-initialization.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import classnames from 'classnames';
+import styles from './styles.css';
+
+const object = {};
+
+for (const key in object) {
+	classnames(styles.foo, styles.bar);
```
Comment (mseeley) [inline test/fixtures/for-in-declaration-in-initialization.js:7]: The repro for the issue doesn't require a `classnames` import or invocation. I considered removing it entirely. Alternatively I could assign the result of the call to `classnames` to a variable. Any changes you'd like to see in this fixture?

**Review by mseeley — COMMENTED** [review]
Pre-review complete.

File: test/fixtures/for-in-declaration-in-initialization.js (hunk 1 above)
Comment (avaly) [reply test/fixtures/for-in-declaration-in-initialization.js:7]: No, the fixture is fine as it is. Thanks!

**Comment by avaly** [comment]
Thanks a lot for finding the bug and fixing it! :tada: :100: :+1:

**Pull request merged**